README.md

```diff
@@ -29,9 +30,21 @@ In your `apollo.config.js` file:
 
 ```js
 const { config } = require("vscode-apollo-relay").generateConfig()
+// If you want to debug the config, uncomment the following line
+// console.log(JSON.stringify(config))
+// You have to restart Visual Code after installing and check the output of Apollo GraphQL (next to the Terimal window)
```

Comment:
I think this line would probably work better as its own paragraph outside of the code block and perhaps include a screenshot that shows the output pane, rather than trying to describe it with “next to the Terminal window”. What do you think?